subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/SwiftXCTestBundle.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.nativeplatform.test.xctest;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.file.RegularFileVar;
+import org.gradle.language.swift.SwiftComponent;
+
+/**
+ * Configuration for a Swift XCTest Bundle, defining the source files that make up the bundle plus other settings.
+ *
+ * <p>An instance of this type is added as a project extension by the Swift XCTest plugin.</p>
```

Comment:
Javadoc style guides suggest not closing tags which don't require closing like `</p>`

Reply:
You can ignore this one ☝️.  I see that we are not following that style in many places in our codebase.